framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/OpenShiftOperatorScenario.java

```diff
@@ -108,6 +118,25 @@ private void createRoleBindingsInProject(Project project) {
         project.getOpenShiftAdmin().rbac().roleBindings().inNamespace(project.getName()).create(roleBinding);
     }
 
+    private void createClusterRoleInProject(Project project) {
+        logger.info("Creating cluster role in project '" + project.getName() + "' from " + OpenShiftResource.CLUSTER_ROLE.getResourceUrl().toString());
+        ClusterRole role = project.getOpenShiftAdmin().rbac().clusterRoles().load(OpenShiftResource.CLUSTER_ROLE.getResourceUrl()).get();
+        project.getOpenShiftAdmin().rbac().clusterRoles().inNamespace(project.getName()).createOrReplace(role);
```

Comment:
Cluster role is shared and if this role already exists, then is thrown exception from Kubernetes client.
```suggestion
        try {
            project.getOpenShiftAdmin().rbac().clusterRoles().inNamespace(project.getName()).create(clusterRole);
        } catch (KubernetesClientException e) {
            if (e.getMessage().contains("already exists")) {
                logger.warn("Cluster role " + OpenShiftResource.CLUSTER_ROLE.getResourceUrl().toString() + " already exists.");
            } else {
                throw new RuntimeException("Exception when creating cluster role from " + OpenShiftResource.CLUSTER_ROLE.getResourceUrl().toString(), e);
            }
        }
```

Reply:
As discussed, we are using the createOrReplace method instead of catching up exceptions.